ui/src/components/entity/EntityAnnotationLayer.tsx

```diff
@@ -41,6 +41,7 @@ interface Props {
   citationAnnotationsEnabled: boolean;
   termAnnotationsEnabled: boolean;
   symbolUnderlineMethod: SymbolUnderlineMethod;
+  definitionsInSymbolGloss: boolean;
```

Comment:
At first, I read this variable as "definitions in symbols, gross"...which felt apt.